src/webportal/src/app/user/fabric/userView/index.jsx

```diff
@@ -0,0 +1,276 @@
+      isOpen: true,
+      innerHtml: userEditModalComponent({
+        'username': user.username,
+        'isAdmin': user.admin,
+        'vcList': user.virtualCluster,
+        'hasGithubPAT': String(user.hasGithubPAT),
+      }),
+    });
+  };
+
+  const hideEditUser = () => {
+    setShowEditInfo({isOpen: false, innerHtml: ''});
+  };
+
+  const updateUserInfoCallback = (data) => {
+    if (data.error) {
+      showMessageBox(data.message);
+    } else {
+      showMessageBox({
+        text: 'Update user information successfully',
+        dismissedCallback: () => {
+          setShowEditInfo({isOpen: false, innerHtml: ''});
+          setAllUsers([]);
+          refreshAllUsers();
+        },
+      });
+    }
+  };
+
+  const updateUserVc = (username) => {
+    const virtualCluster = $('#form-update-virtual-cluster :input[name=virtualCluster]').val();
+    updateUserVcRequest(username, virtualCluster)
+      .then(updateUserInfoCallback)
+      .catch((err) => {
+        $('#form-update-virtual-cluster').trigger('reset');
+        showMessageBox(err);
+      });
+  };
+
+  const updateUserAccount = (username) => {
+    const password = $('#form-update-account :input[name=password]').val();
+    const admin = $('#form-update-account :input[name=admin]').is(':checked') ? true : false;
+    updateUserAccountRequest(username, password, admin)
+      .then(updateUserInfoCallback)
+      .catch((err) => {
+        $('#form-update-account').trigger('reset');
+        showMessageBox(err);
+      });
+  };
+
+  const updateUserGithubPAT = (username) => {
+    const githubPAT = $('#form-update-github-token :input[name=githubPAT]').val();
+    updateUserGithubPATRequest(username, githubPAT)
+      .then(updateUserInfoCallback)
+      .catch((err) => {
+        $('#form-update-github-token').trigger('reset');
+        showMessageBox(err);
+      });
+  };
+
+  window.updateUserVc = updateUserVc;
+  window.updateUserAccount = updateUserAccount;
+  window.updateUserGithubPAT = updateUserGithubPAT;
+  window.hideEditUser = hideEditUser;
+
+  const context = {
+    allUsers,
+    refreshAllUsers,
+    filteredUsers,
+    ordering,
+    setOrdering,
+    filter,
+    setFilter,
+    pagination,
+    setPagination,
+    setSelectedUsers,
+    getSelectedUsers,
+    setAllSelected,
+    addUser,
+    importCSV,
+    removeUsers,
+    editUser,
+  };
+
+  const {spacing} = getTheme();
+
+  return (
+    <Context.Provider value={context}>
+      <Fabric className={t.h100}>
+        <Stack verticalFill styles={{root: [t.relative]}}>
+          <Stack.Item>
+            <TopBar />
+          </Stack.Item>
+          <Stack.Item grow styles={{root: [t.overflowAuto, t.bgWhite, {paddingTop: spacing.m}]}}>
+            <Table />
+          </Stack.Item>
+          <Stack.Item>
+            <Paginator />
+          </Stack.Item>
+        </Stack>
+      </Fabric>
+      <Modal
+        isOpen={showEditInfo.isOpen}
+        styles={{main: [t.mw8, t.w90]}}>
```

Comment:
Tachyons uses rem by default, but AdminLTE will add a global font size to override the browser's font setting.
So I suggest uses pixels here.

Reply:
Good suggestion. Fixed.